clients/voicechat/nemotron-voicechat-client.py

```
@@ -0,0 +1,1648 @@
+            self.conversation_log.append(
+                {
+                    "role": "agent",
+                    "content": "".join(self.current_agent_text),
+                    "timestamp": time.time() - self.start_time,
+                }
+            )
+
+        # Save function call log
+        try:
+            function_filename = (
+                self.function_text_output if self.function_text_output else f"function_calls.jsonl"
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)